language/ar/upload.php

```diff
@@ -0,0 +1,206 @@
+		'solution'	=> 'نرجوا التأكد من صحة الملف المضغوط والمحاولة مرة أخرى.',
+	),
+	'NOT_AN_EXTENSION'					=> 'لم يتم التعرف على الملف المضغوط الذي رفعته. وبالتالي لم يتم حفظ هذا الملف على سيرفرك.',
+	'EXT_ACTION_ERROR'					=> 'لا يمكن تنفيذ الإجراء المطلوب للإضافة التي حددتها.<br />ملاحظة : لا يمكن إدارة الإضافة "رفع الإضافات" إلا بواسطة إدارة الإضافات القياسي.',
+
+	'SOURCE'							=> 'المصدر',
+	'EXTENSION_UPDATE_NO_LINK'			=> 'رابط التحميل غير متوفر.',
+	'EXTENSION_TO_BE_ENABLED'			=> 'سوف يتم تعطيل "رفع الإضافات" أثناء عملية التحديث وإعادة التفعيل بعد التحديث.',
+	'EXTENSION_UPLOAD_UPDATE'			=> 'تحديث الإضافة',
+	'EXTENSION_UPLOAD_UPDATE_EXPLAIN'	=> 'سيتم تنفيذ عملية الرفع من الرابط المبين أدناه.',
+
+	'EXTENSION_UPLOADED'				=> 'تم رفع الإضافة “%s” بنجاح.',
+	'EXTENSIONS_AVAILABLE'				=> 'الإضافات الغير مُثبتة',
+	'EXTENSIONS_UPLOADED'				=> 'الإضافات المرفوعة',
+	'EXTENSIONS_UNAVAILABLE'			=> 'الإضافات الغير صالحة',
+	'EXTENSIONS_UNAVAILABLE_EXPLAIN'	=> 'تم رفع الإضافات المذكورة في القائمة أدناه إلى منتداك ولكنها لم تعد صالحة بسبب بعض العوامل , ولذلك هي غير متوفرة ولا يمكن تفعيلها في منتداك. نرجوا التأكد من وجود الملفات الخاصة بهذه الإضافات بمكانها الصحيح أو استخدم أداة تنظيف الإضافات لو رغبت في حذف هذه الإضافات الغير صالحة من السيرفر لديك.',
+	'EXTENSION_BROKEN'					=> 'إضافة غير صالحة',
+	'EXTENSION_BROKEN_ENABLED'			=> 'تم تفعيل هذه الإضافة الغير صالحة !',
+	'EXTENSION_BROKEN_DISABLED'			=> 'تم تعطيل هذه الإضافة الغير صالحة !',
+	'EXTENSION_BROKEN_TITLE'			=> 'هذه الإضافة غير صالحة !',
+	'EXTENSION_BROKEN_DETAILS'			=> 'انقر هنا لقراءة التفاصيل.',
+	'EXTENSION_BROKEN_EXPLAIN'			=> '<strong>لا تزال بعض بيانات هذه الإضافة مخزونة في السيرفر.</strong> الرجاء التأكد من سبب عدم عمل هذه الإضافة وكذلك من وجود ملفاتها في المسار المطلوب. قد تحتاج طلب المساعدة من مبرمجي الإضافة واستخدام برنامج نقل البيانات FTP لتغيير بعض الملفات ( أو تستطيع رفع نسخة جديدة بالإصلاحات ). عند ذلك سوف تستطيع إدارة هذه الإضافة مرة أخرى.<br /><h3>تستطيع عمل التالي :</h3><br /><strong>تحديث الإضافة الغير صالحة.</strong><br /><ul><li>التأكد من تعطيل الإضافة ( انقر على زر التعطيل لو أستدعى الأمر ).</li><li>التأكد من توفر إصدار جديد للإضافة ورفعها إلى المسار الصحيح.</li><li>طلب المساعدة من مبرمجي الإضافة إذا لم يتم حل المشكلة بالخطوات أعلاه.</ul><strong>أو</strong><br /><br /><strong>إزالة الإضافة الغير صالحة بالكامل.</strong><br /><ul><li>التأكد من تعطيل الإضافة ( انقر على زر التعطيل لو أستدعى الأمر ).</li><li>التأكد من حذف البيانات للإضافة ( انقر على زر الحذف لو أستدعى الأمر ).</li><li>احذف ملفات الإضافة بواسطة أداة حذف الإضافات.</ul>',
+
+	'EXTENSION_UPLOADED_ENABLE'			=> 'تفعيل هذه الإضافة',
+	'ACP_UPLOAD_EXT_UNPACK'				=> 'فك الضغط',
+	'ACP_UPLOAD_EXT_CONT'				=> 'محتوى المجلد : %s',
+
+	'EXTENSION_DELETE'					=> 'حذف الإضافة',
+	'EXTENSION_DELETE_CONFIRM'			=> 'هل تريد بالفعل حذف الإضافة “%s” ؟',
+	'EXTENSIONS_DELETE_CONFIRM'			=> array(
+		2	=> 'هل تريد بالفعل حذف عدد <strong>%1$s</strong> إضافات ؟',
```

Comment:
@alhitary, I think you should read this guideline: https://wiki.phpbb.com/Plural_Rules.
Arabic language has plural rule number 12. Please add other lines as described [here](https://wiki.phpbb.com/Plural_Rules#How_to_use_the_rules).

This comment also applies to other occurrences of this rule. I'll add a note about that to language files later.